randomizer/randomize.py

```diff
@@ -687,5 +685,5 @@ def randomize(rom_data: bytearray, seed: str, flags: Flags) -> bytearray:
     all_patches.update(pack_vehicle_starts(vehicle_starts))
 
     randomized_rom = rom.apply_patches(all_patches)
-
+    print("Randomization Finished")
```

Comment:
Can you change this to print(f"...") and add in the seed & flags strings?

Reply:
I figured that since we had the in the initial print it wasn't necessary?

Can still add if you want.

Reply:
Oh! Huh! Okay, yeah, then this is fine. Ooops!